src/OpenLoco/src/CommandLine.cpp

```diff
@@ -375,7 +376,7 @@ namespace OpenLoco
         std::cout << "                host [options] <path>" << std::endl;
         std::cout << "                join [options] <address>" << std::endl;
         std::cout << "                uncompress [options] <path>" << std::endl;
-        std::cout << "                simulate [options] <path> <ticks>" << std::endl;
+        std::cout << "                simulate [options] <path> <ticks> [path]" << std::endl;
```

Comment:
i would probably just make it a standalone command. I know that i'll mostly be using it with a simulate but it might be useful to be able to compare saves without a simulate.